cmd/brew-report-issue.rb

```diff
@@ -0,0 +1,105 @@
+#!/usr/bin/env ruby
+# Creates and closes issues on a project.
+close = !!ARGV.delete("--close")
+user_repo = ARGV.shift
+failure_source = ARGV.shift
```

Comment:
It might be a little clearer if you define `failure_source` only if `!close`, and `success_meta if it was close, ie:

```ruby
if close
  failure_source = ARGV.shift
else
  success_meta = ARGV.shift
end
```